### DISCUSSION TITLE
A Discord Community for QuickJS

### DISCUSSION THREAD

**Discussion by ammarahm-ed** [quickjs-ng/quickjs#117] [Q&A] [answered]
Creating a Discord community for QuickJS where people interested in the project can come and chat about it would be awesome.

**Comment by saghul** [comment]
I am heavily -1 on using closed communities using proprietary software.

GH discussions sounds like a good way to start and we could take it from there.

Matrix is a good option in the more open space and before Ben mentions it, good old IRC ;-)

**Reply by ammarahm-ed** [reply]
Anywhere one could just chat on a phone would be way nicer to discuss ideas or help each other out. Good thing about discord is a lot of the oss community is already there I believe, but anything will work.

**Comment by M0n7y5** [comment]
Discord is heavily adopted by other OSS communities. It has an amazing tooling system and a really good permission system. It offers more than simple IRC like chats. Forums, threads, events and more.

**Reply by saghul** [reply]
It is still not open source and all content would be gated and not consumable by non members.

This very discussion feature of GH does offer what we need, for now.

Matrix is a good option for instant messaging like setups.

**Comment by BigJk** [comment]
Great to see quickjs getting more love and I would really like to join the community on whatever platform is chosen. I think having a instant messaging solution (Discord, Matrix, I don't care) does have some benefits in contrast to only using discussions

**Reply by saghul** [reply, marked as answer]
Let's give it a try: https://matrix.to/#/!ksanNyErnjTFITiPlP%3Amatrix.org

Or #quickjs-ng:matrix.org
